commands/ban.js

```diff
@@ -1,49 +1,73 @@
-const embed = require("../utils/embed.js");
+const { Permissions } = require('discord.js');
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const embed = require('../utils/embed.js');
 
 module.exports = {
-	name: "ban",
-	description: "Bans a very naughty user.",
-	usage: "[@user]",
-	cooldown: 0,
-	guildOnly: true,
-	modOnly: true,
```

Comment:
I personally think that this config driven approach is quite nice, unless there's a big reason not too I would keep it and have slash command definitions built based upon it. Would help reduce boiler plate on each command

Reply:
We discussed this and the abstraction isn't worth it.